routes/web.php

```diff
@@ -9,58 +19,68 @@
 | routes are loaded by the RouteServiceProvider within a group which
 | contains the "web" middleware group. Now create something great!
 |
-*/
-
-Route::get('/', function () {
-    return view('welcome');
-});
+ */
 
-use App\Models\Concept;
-use Illuminate\Http\Request;
-use Illuminate\Support\Facades\Auth;
-
-Route::get('concepts',              'ConceptController@index');
-// Route::post('concepts',             'ConceptController@store')->middleware('can:edit-vocabulary'); // TODO: switch to can:edit-vocabulary after demo testing
-Route::post('concepts',             'ConceptController@store')->middleware('auth');
-Route::get('concepts/create',       'ConceptController@create')->middleware('auth');
-Route::get('concepts/search',       'ConceptController@search')->name('search');
-Route::get('concepts/search_page',       'ConceptController@search_page');
-Route::post('concepts/{concept}/add_term', 'ConceptController@addTerm')->middleware('auth');
-Route::get('concepts/{concept}',    'ConceptController@show');
-Route::delete('concepts/{concept}', 'ConceptController@destroy');
+#=====================================
+# Authentication
+#=====================================
+// Disable Laravel user registration
+Auth::routes(['register' => false]);
 
 Route::post('logout/all', function () {
     return Redirect::away(env('SNAC_AUTHENTICATION_URL') . '?command=logout3');
 });
 Route::get('logoff', function () {
     Auth::logout();
-    if(isset($_GET['redirect'])) {
+    if (isset($_GET['redirect'])) {
         return redirect(urldecode($_GET['redirect']));
     }
     return redirect('/');
 });
 
-Route::get('login/snac', function() {
+Route::get('login/snac', function () {
     session(['_from_snac' => true]);
-    if(isset($_GET['redirect'])) {
+    if (isset($_GET['redirect'])) {
         session(['_redirect_after_login' => $_GET['redirect']]);
     }
     return Socialite::driver('google')->redirect();
 });
 
-Route::get('login/github', 'Auth\LoginController@redirectToGitHubProvider');
-Route::get('github/login', 'Auth\LoginController@handleGitHubProviderCallback');
+Route::get('login/github', [LoginController::class, 'redirectToGitHubProvider']);
+Route::get('github/login', [LoginController::class, 'handleGitHubProviderCallback']);
```

Comment:
I'm surprised to see github login. We can probably drop these two lines unless they're useful for development.